src/views/CVERecord/PublishedRecord.vue

```diff
@@ -1,338 +1,119 @@
-            <div id="cve-platforms" class="pt-3" v-if="product.platforms.length > 0">
-              <p class="has-text-weight-bold mb-1">Platforms</p>
-              <p>{{product.platforms.join(', ')}}</p>
+          </nav>
+          <hr/>
+          <nav class="level mb-0 is-mobile">
+            <div class="level-left">
             </div>
-          </div>
-          <div class="column cve-left-margin cve-top-margin cve-light-gray-bg">
-            <p class="has-text-weight-bold mb-1">Versions</p>
-            <p v-if="product.defaultStatus" class="cve-help-text is-italic">
-              <span class="has-text-weight-bold">Default Status: </span>{{product.defaultStatus}}
-            </p>
-            <p v-else class="cve-help-text is-italic"><span class="has-text-weight-bold">Default Status: </span>unknown</p>
-            <div class="columns mr-1 mb-1 cve-versions-columns">
-              <!-- Loop through the column headers so the affected/unaffected/unknown columns will display in alphabetical order -->
-              <div v-for="header in product.versionsColumns.headers" :key="header.key"
-                class="column pt-0 pr-0 pl-1 cve-left-margin cve-top-margin cve-versions-column">
-                <div class="menu p-0" style="border: unset !important">
-                  <ul class="menu-list mt-1" v-for="row in product.versionsColumns.twoDTable[header]" :key="row.key">
-                    <li>
-                      <span v-if="row.versionsChanges.length > 0">
-                        <p>
-                          {{row.parentVersionStatus}}
-                          <span v-for="word in row.parentVersionRange" :key="word.key" :class="isAversion(word) ? 'cve-version' : ''">{{word}} </span>
-                        </p>
-                        <ul style="border-left: none !important;">
-                          <li v-for="change in row.versionsChanges" :key="change.key">
-                            <span v-for="word in change" :key="word.key" :class="isAversion(word) ? 'cve-version' : ''">{{word}} </span>
-                          </li>
-                        </ul>
-                      </span>
-                      <span v-else>
-                        <span v-if="row.parentVersionRange.length > 0">{{row.parentVersionStatus}}
-                          <span v-for="word in row.parentVersionRange" :key="word.key" :class="isAversion(word) ? 'cve-version' : ''">{{word}} </span>
-                        </span>
-                        <span v-else>
-                          {{row.parentVersionStatus}} {{row.parentVersionRange}}
-                        </span>
-                      </span>
-                    </li>
-                  </ul>
+            <div class="level-right ml-1">
+              <div class="level-item">
+                <div id="cve-accordian-expand-collapse-all-btn" class="buttons mb-0" style="justify-content: flex-end;">
```

Comment:
Can you try the Bulma class `is-align-content-flex-end` here instead of the inline style? I think there are other classes that might work, too